python/cuml/common/input_utils.py

```diff
@@ -105,6 +108,14 @@ def input_to_cuml_array(X, order='F', deepcopy=False,
         Set to True if you want the method to raise a ValueError if X is not
         of order `order`.
 
+    force_contiguous: boolean (default: True)
+        Set to True to force CumlArray produced to be contiguos. If `X` is
```

Comment:
typo on contiguous

Reply:
I can't type that reliably to save my life :P